eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/zmq/ZMQActor.scala

```diff
@@ -116,7 +117,7 @@ object ZMQActor {
   // @formatter:on
 
   object Topics {
-    val RawBlock: String = "rawblock"
+    val HashBlock: String = "hashblock"
```

Comment:
This feels really weird, should probably be:
```suggestion
    val BlockHash: String = "blockhash"
```

Reply:
Wait, the official topic name is actually `hashblock`, nevermind.